perception/autoware_tensorrt_bevdet/CMakeLists.txt

```diff
@@ -0,0 +1,102 @@
+cmake_minimum_required(VERSION 3.17)
+project(autoware_tensorrt_bevdet)
+
+add_compile_options(-W)
+add_compile_options(-std=c++17)
+set(CMAKE_CXX_FLAGS_RELEASE "-Wno-deprecated-declarations -O2")
+set(CUDA_NVCC_FLAGS "${CUDA_NVCC_FLAGS} -std=c++17 -O0 -Xcompiler -fPIC -g -w -gencode=arch=compute_86,code=sm_86")
```

Comment:
Consider using a more descriptive variable name instead of `-W` for the compiler warning flag.  For example, `-Wall` (enables all warnings) or a more specific flag like `-Wextra` might be more informative.  This improves readability and clarifies the intended level of warning checking.